pages/common/enscript.md

```diff
@@ -0,0 +1,23 @@
+# enscript
+
+> A tool to convert text files to PostScript, HTML, RTF, ANSI, and overstrikes.
+
+- Generate PostScript from a file and output to another:
+
+`enscript {{path/to/input_file}} --output={{path/to/output_file}}`
+
+- Generate a certain output language (eg. _html_) from a file and output to another:
```

Comment:
html should not be in underscores. Rather "html" is fine

Reply:
With or without quotation marks?

Reply:
Usually we would have used quotes. But the way you have constructed the sentence, even without quotes is fine

Reply:
Ok, but for consistency and clarity I will include them.